lib/api-base/fastApiContainer.ts

```diff
@@ -35,97 +35,97 @@ const CONTAINER_MEMORY_BUFFER = 1024 * 2;
+    /** FastAPI container */
+    public readonly container: ContainerDefinition;
 
-  /** FastAPI IAM task role */
-  public readonly taskRole: IRole;
+    /** FastAPI IAM task role */
+    public readonly taskRole: IRole;
 
-  /** FastAPI URL **/
-  public readonly endpoint: string;
+    /** FastAPI URL **/
+    public readonly endpoint: string;
 
-  /**
+    /**
    * @param {Construct} scope - The parent or owner of the construct.
    * @param {string} id - The unique identifier for the construct within its scope.
    * @param {RestApiProps} props - The properties of the construct.
    */
-  constructor(scope: Construct, id: string, props: FastApiContainerProps) {
-    super(scope, id);
-
-    const { config, securityGroup, taskConfig, tokenTable, vpc } = props;
-
-    let buildArgs: Record<string, string> | undefined = undefined;
-    if (taskConfig.containerConfig.image.type === EcsSourceType.ASSET) {
-      buildArgs = {
-        BASE_IMAGE: taskConfig.containerConfig.image.baseImage,
-        PYPI_INDEX_URL: config.pypiConfig.indexUrl,
-        PYPI_TRUSTED_HOST: config.pypiConfig.trustedHost,
-        LITELLM_CONFIG: yamlDump(config.litellmConfig),
-      };
-    }
-    const environment: Record<string, string> = {
-      LOG_LEVEL: config.logLevel,
-      AWS_REGION: config.region,
-      AWS_REGION_NAME: config.region, // for supporting SageMaker endpoints in LiteLLM
-      THREADS: Ec2Metadata.get(taskConfig.instanceType).vCpus.toString(),
-      LITELLM_KEY: config.litellmConfig.general_settings.master_key, // for model management operations
-    };
-
-    if (config.restApiConfig.internetFacing) {
-      environment.USE_AUTH = 'true';
-      environment.AUTHORITY = config.authConfig!.authority;
-      environment.CLIENT_ID = config.authConfig!.clientId;
-    } else {
-      environment.USE_AUTH = 'false';
-    }
-
-    if (tokenTable) {
-      environment.TOKEN_TABLE_NAME = tokenTable.tableName;
+    constructor (scope: Construct, id: string, props: FastApiContainerProps) {
+        super(scope, id);
+
+        const { config, securityGroup, taskConfig, tokenTable, vpc } = props;
+
+        let buildArgs: Record<string, string> | undefined = undefined;
+        if (taskConfig.containerConfig.image.type === EcsSourceType.ASSET) {
+            buildArgs = {
+                BASE_IMAGE: taskConfig.containerConfig.image.baseImage,
+                PYPI_INDEX_URL: config.pypiConfig.indexUrl,
+                PYPI_TRUSTED_HOST: config.pypiConfig.trustedHost,
+                LITELLM_CONFIG: yamlDump(config.litellmConfig),
+            };
+        }
+        const environment: Record<string, string> = {
+            LOG_LEVEL: config.logLevel,
+            AWS_REGION: config.region,
+            AWS_REGION_NAME: config.region, // for supporting SageMaker endpoints in LiteLLM
+            THREADS: Ec2Metadata.get(taskConfig.instanceType).vCpus.toString(),
+            LITELLM_KEY: config.litellmConfig.general_settings.master_key, //
```

Comment:
nit: can remove trailing `//`

Reply:
+1 or we can keep the comment. either way is fine for me.